src/presence_analyzer/tests.py

```diff
@@ -86,8 +144,73 @@ def test_get_data(self):
             datetime.time(9, 39, 5)
         )
 
+    def test_seconds_since_midnight(self):
+        """
+        Test time elapsed with different time set. Special case: midnight.
+        """
+        self.assertEqual(utils.seconds_since_midnight(datetime.time(0, 0, 0)), 0)
+        self.assertEqual(utils.seconds_since_midnight(datetime.time(0, 0, 1)), 1)
+        self.assertEqual(utils.seconds_since_midnight(datetime.time(0, 1, 0)), 60)
+        self.assertEqual(utils.seconds_since_midnight(datetime.time(1, 0, 0)), 3600)
+
+    def test_interval(self):
+        """
+        Test interval is counted correctly for two different and two same points in time.
+        """
+        self.assertEqual(utils.interval(datetime.time(0, 0, 0), datetime.time(0, 0, 0)), 0)
+        self.assertEqual(utils.interval(datetime.time(0, 0, 0), datetime.time(1, 1, 1)), 3661)
+
+    def test_mean_with_empty_list(self):
+        """
+        Mean checks whether provided list is empty and returns 0 if so.
+        """
+        self.assertEqual(utils.mean([]), 0)
+
+    def test_mean_with_non_empty_list(self):
+        """
+        Count mean and assert it provides 3-point accuracy.
+        """
+        self.assertAlmostEqual(utils.mean([1, 2]), 1.5, 3)
```

Comment:
`test_mean_with_empty_list` and `test_mean_with_non_empty_list` in one test
Couple more cases.